utils/python/CIME/case.py

```diff
@@ -411,6 +412,24 @@ def configure(self, compset_name, grid_name, machine_name=None,
         for key, value in pes_rootpe.items():
             mach_pes_obj.set_value(key,int(value))
 
+        for compclass in self._component_classes:
```

Comment:
Make sure that config_pes.xml has accounted for every component present and if not set the nthrds and ntasks values for that component to 1